boards/common/atxmega/include/periph_conf_atxmega_common.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_atxmega
+ * @{
+ *
+ * @file
+ * @brief       Common configuration of MCU periphery for ATxmega boards
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+
+#ifndef PERIPH_CONF_ATXMEGA_COMMON_H
```

Comment:
There's no need to add atxmega in the header file name: periph_conf_common.h is ok because this file is already in a subdirectory  called atxmega